### PR TITLE
Add CF aliases

### PR DESCRIPTION
**Pull request by tlimoncelli**

### CODE REVIEW

File: docs/_providers/cloudflare.md [hunk 1]
```diff
@@ -30,6 +30,37 @@ Domain level metadata availible:
 Provider level metadata availible:
    * ip_conversions
 
+Note: Aliases are pre-defined as follows:
+
+{% highlight json %}
+var CF_PROXY_OFF = {'cloudflare_proxy': 'off'};     // Default/off.
+var CF_PROXY_ON = {'cloudflare_proxy': 'on'};       // Sites safe to proxy.
+var CF_PROXY_FULL = {'cloudflare_proxy': 'full'};   // Sites safe to railgun.
+var SET_PROXY_DEFAULT_TRUE = CF_PROXY_ON; // Turn on CF proxy for entire domain.
```
Comment (captncraig) [inline docs/_providers/cloudflare.md]: The comments here are not particularly clear to me.

Comment (captncraig) [inline docs/_providers/cloudflare.md]: We should probably call these CF_PROXY_DEFAULT_(ON|OFF|FULL)

Comment (tlimoncelli) [inline docs/_providers/cloudflare.md]: I've made changes as requested. PTAL

**Comment by tlimoncelli** [comment]
I think this is ready to merge.

**Pull request merged**